chapters/packages.tex

```diff
@@ -208,7 +208,7 @@ \section{Mapping Package/Class Structures to a Hierarchical File System}\label{m
   File in the file system.
 \end{itemize}
 
-Each Modelica file in the file-system is stored in UTF-8 format (defined by The Unicode Consortium; \lstinline!http://www.unicode.org!).  A deprecated feature is that the file may start with the UTF-8 encoded BOM (byte order mark; \lstinline!0xef 0xbb 0xbf!); this is treated as white-space in the grammar.  Since the use of BOM is deprecated, tools can ignore any BOM when reading, and it is recommended to never write it.
+Each Modelica file in the file-system is stored in UTF-8 format (defined by The Unicode Consortium; \url{http://unicode.org}).  A deprecated feature is that the file may start with the UTF-8 encoded BOM (byte order mark; \lstinline!0xef 0xbb 0xbf!); this is treated as white-space in the grammar.  Since the use of BOM is deprecated, tools can ignore any BOM when reading, and it is recommended to never write it.
```

Comment:
Both with and without the www redirect to https://home.unicode.org/ . Should we respect that and link directly there?

Reply:
I prefer the shorter [https://unicode.org](url) as long as it works. It might even be more stable, and I couldn't find any written text saying that one or the other is preferable.

Reply:
You can see it seems to be using a "meta refresh" like this: https://en.wikipedia.org/wiki/URL_redirection#Refresh_Meta_tag_and_HTTP_refresh_header 

If you use curl:

```
maltel@elantris:~$ curl -i https://unicode.org
HTTP/1.1 200 OK
Date: Fri, 03 Dec 2021 12:49:04 GMT
Server: Apache/2.4.38 (Debian)
Content-Security-Policy: upgrade-insecure-requests;
Last-Modified: Wed, 17 Jul 2019 13:08:41 GMT
ETag: "e3-58de031306840"
Accept-Ranges: bytes
Content-Length: 227

<html><head>
<meta http-equiv="refresh" content="0; url=http://home.unicode.org/">
<title>Index</title>
</head>
<body>
<!-- Automatic redirect: <a href="http://home.unicode.org/">http://home.unicode.org/</a>
-->
</body></html>
```

According to the Wikipedia page:
> This is treated like a 301 permanent redirect by Google

So we should probably do the same and just point to the end result?

Reply:
I agree with https - but I thought the 301 handling by google was primarily for their search-results, since they downgrade temporary stuff. So it is "permanent" in the sense that isn't just happening this week, but whether they will change it the next year is unclear.

Reply:
You might be right. Either way I don't feel strongly enough about it to force the issue, so resolving.